jfxtras-controls/src/main/java/jfxtras/internal/scene/control/skin/CalendarTextFieldSkin.java

```diff
@@ -67,6 +67,13 @@
  */
 public class CalendarTextFieldSkin extends SkinBase<CalendarTextField>
 {
+    
+    private TextField textField = null;
+    private ImageView imageView = null;
+    private GridPane gridPane = null;
+    final public BooleanProperty focusForwardingProperty = new SimpleBooleanProperty();
+    private final Popup popup = new Popup();
```

Comment:
I put the popup here because I needed to listen to the showingProperty of the Popup and it was easier to put it once.
I'll revert to the old way where it's instantiated on demand and add the listener when necessary.